README.md

```diff
@@ -1 +1,2 @@
 Automating AWS with Terraform
+We are using gitbranches to track changes
```

Comment:
README.md text gitbranches not resolved yet

Reply:
add space between the text gitbranches to become git branches

Reply:
Please can you advice me on what to do next, I have made 3 commits already

Reply:
Check your README.md file change the text gitbranches to git branches then run
 git add README.md
 and then commit and push your changes